// postcss-plugin-utilities/index.js
module.exports = {
    calc:                   require('./inc/calc.js'),
    contrastColor:          require('./inc/contrast-color.js'),
    eachSelector:           require('./inc/each-selector.js'),
    filterObject:           require('./inc/filter-object.js'),
    getRGB:                 require('./inc/get-rgb.js'),
    getSides:               require('./inc/get-sides.js'),
    hexToRGB:               require('./inc/hex-to-rgb.js'),
    isBezier:               require('./inc/is-bezier.js'),
    isBorder:               require('./inc/is-border.js'),
    isBoxShadow:            require('./inc/is-box-shadow.js'),
    isColor:                require('./inc/is-color.js'),
    isCursor:               require('./inc/is-cursor.js'),
    isHTML:                 require('./inc/is-html.js'),
    isNumber:               require('./inc/is-number.js'),
    isProperty:             require('./inc/is-property.js'),
    isRegex:                require('exact-regex'),
    isSizeList:             require('./inc/is-size-list.js'),
    isSize:                 require('./inc/is-size.js'),    
    isStep:                 require('./inc/is-step.js'),
    isTextShadow:           require('./inc/is-text-shadow.js'),
    isTime:                 require('./inc/is-time.js'),
    isTransition:           require('./inc/is-transition.js'),
    isURL:                  require('./inc/is-url.js'),    
    nameToHex:              require('./inc/name-to-hex.js'),
    overwrite:              require('overwrite-object'),
    parseFunction:          require('./inc/parse-function.js'),
    parseValue:             require('postcss-value-parser'),
    removeNode:             require('./inc/remove-node.js'),
    rgbToHex:               require('./inc/rgb-to-hex.js'),
    sassHasVar:             require('./inc/sass-has-var.js'),
    sassGetVar:             require('./inc/sass-get-var.js')
}
